sentry_sdk/integrations/falcon.py

```diff
@@ -0,0 +1,162 @@
+from typing import Any, Callable, Dict
+
+import falcon
+import falcon.api_helpers
+import sentry_sdk.integrations
+from sentry_sdk.hub import Hub
+from sentry_sdk.integrations._wsgi_common import RequestExtractor
+from sentry_sdk.integrations.wsgi import SentryWsgiMiddleware
+from sentry_sdk.utils import capture_internal_exceptions, event_from_exception
+
+
+class FalconRequestExtractor(RequestExtractor):
+
+    def env(self):
+        return self.request.env
+
+    def cookies(self):
+        return self.request.cookies
+
+    def raw_data(self):
+        # As request data can only read once we won't make this available
+        # to Sentry.
+        # TODO(jmagnusson): Figure out if there's a way to support this
+        return None
+
+    def form(self):
+        return None  # No such concept in Falcon
+
+    def files(self):
+        return None  # No such concept in Falcon
+
+    def json(self):
+        # We don't touch falcon.Request.media as that can raise an exception
+        # on non-JSON requests.
+        return self.request._media
+
+
+class SentryFalconMiddleware(object):
+    """Captures exceptions in Falcon requests and send to Sentry"""
+
+    def process_request(self, req, resp, *args, **kwargs):
+        hub = Hub.current
+        integration = hub.get_integration(FalconIntegration)
+        if integration is None:
+            return
+
+        with hub.configure_scope() as scope:
+            scope.add_event_processor(
+                _make_request_event_processor(req, integration)
+            )
+
+
+class FalconIntegration(sentry_sdk.integrations.Integration):
```

Comment:
Could you rewrite this import to import `Integration` into this namespace? (like other integrations)

Reply:
Fixed